### DISCUSSION TITLE
How to get the Active users data in Plausible?

### DISCUSSION THREAD

**Discussion by Pradeeppon01** [plausible/analytics#3964] [Self-Hosted Support]
In the Plausible dashboard, I can observe the total count of active users, but I'm unable to identify the specific users who are currently active. Is there a method available to access detailed information about the active users on the site?

**Comment by ruslandoga** [comment]
👋 @Pradeeppon01 

There is no way to do that right now. It's privacy-friendly analytics after all.

**Reply by digital88** [reply]
There is a way actually, you can attach custom props to events: https://plausible.io/docs/custom-props/for-pageviews
Later you can query Clickhouse for this custom props.

Custom prop can be your user_id or whatever.

Obv this is valid only for selfhost setup.
